Framework/Backend/services/jira.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+*/
+
+const https = require('https');
+
+/**
+ * Handles creating JIRA issues
+ */
+class Jira {
+  /**
+   * @param {object} config - JIRA configuration including URL, service account and project ID
+   */
+  constructor(config) {
+    this.url = config.url;
+    this.accountUser = config.accountUser;
+    this.accountPass = config.accountPass;
+    this.projectId = config.projectId;
+    this.issueTypes = {
+      bug: 1
+    };
+  }
+
+  /**
+   * Handles HTTP req/rep in order to create JIRA issue
+   * @param {object} postData - issue details
+   */
+  async createIssue(postData) {
+    const requestOptions = {
+      method: 'POST',
+      auth: this.accountUser + ':' + this.accountPass,
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+        'Content-Length': Buffer.byteLength(postData)
+      }
+    };
+    return new Promise((resolve, reject) => {
```

Comment:
Not sure if I understood reason behind it, could you explain?

Reply:
I was suggesting that the functionality related to the HTTP request can be extracted in a separate function so that in the future it can be reused by a potential `getTickets` method or something similar.